src/Core/Scanner.vala

```diff
@@ -28,14 +28,17 @@ namespace PantheonCalculator.Core {
         public unichar[] uc;
 
         private unichar decimal_symbol;
+	private unichar separator_symbol;
 
         public Scanner (string str) {
```

Comment:
Can you please convert the class so it uses [GObject construction](https://chebizarro.gitbooks.io/the-vala-tutorial/content/gobject-style-construction.html)?